src/main/java/seedu/address/ui/TaskCard.java

```diff
@@ -0,0 +1,76 @@
+package seedu.address.ui;
+
+import javafx.beans.binding.Bindings;
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.task.ReadOnlyTask;
+
+//@@author Esilocke
+/**
```

Comment:
Structure of TaskCard and TaskListPanel look similar to the Person ones. This should make the font size implementation easier. Thanks!